net/ztm/Makefile

```diff
@@ -0,0 +1,68 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ztm
+PKG_VERSION:=1.5.9
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/flomesh-io/pipy.git
+PKG_MIRROR_HASH:=bab44ce1fa8aff613e71fee6b3e0621163764df97739c41175af01511a1ca3a8
+PKG_SOURCE_VERSION:=ddd5f55915eaca575ac782c4a12c0e3110d3d5e2
+PKG_SOURCE_DATE:=2025-01-16
+PKG_MAINTAINER:=Flomesh <contact@flomesh.io>
```

Comment:
This should be a real (i.e. your) name as per [contributing guidelines](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain) and should match the `Signed-off-by` in case of a new package.